ci-operator/config/redhat-openshift-ecosystem/cvp/redhat-openshift-ecosystem-cvp-ocp-4.6.yaml

```diff
@@ -11,11 +11,57 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: cvp-common-claim
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.6"
+  cron: '@yearly'
+  steps:
+    dependency_overrides:
+      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-3
+      OO_INDEX: brew.registry.redhat.io/rh-osbs-stage/iib:23576
+    env:
+      OO_CHANNEL: "4.3"
+      OO_INSTALL_NAMESPACE: '!create'
+      OO_PACKAGE: e2e-test-operator
+      OO_TARGET_NAMESPACES: '!install'
+      PYXIS_URL: ""
+    workflow: optional-operators-cvp-common-claim
 - as: cvp-common-aws
   cron: '@yearly'
   steps:
     cluster_profile: aws-cpaas
     workflow: optional-operators-cvp-common-aws
+- as: cvp-common-powervs
```

Comment:
The `powervs`, `probe` and `ibmz` job are out of scope for this ticket and the support for these jobs doesn't exist now - I propose removing this entire section down to line 64